pros/serial/devices/vex/v5_device.py

```diff
@@ -947,7 +949,7 @@ def _rx_ext_packet(cls, msg: Message, command: int, rx_length: int, check_ack: b
         if len(msg) > 0:
             logger(cls).debug('Set msg window to {}'.format(bytes_to_str(msg)))
         if len(msg) < rx_length and check_length:
-            raise VEXCommError(f'Received length is less than {rx_length} (got {len(msg)})', msg)
+            raise VEXCommError(f'Received length is less than {rx_length} (got {len(msg)}). Consider updating vexOS.', msg)
```

Comment:
you know, i don't think that's what this signifies. far more likely that some data was lost in the aether (we already check for version above and i think we've been told they'll never remove stuff from the schema)

Reply:
Fair enough, most of the time I see this when there's something physically wrong with the programming cable. Thought this might stop people from asking why the new pros cli version isn't working with their old vexOS but it might also lead to people asking more questions in general when data is actually lost

Reply:
yeah that's sort of my train of thought. not that we dislike people asking questions, of course 🙂 just don't want to start people down the wrong road and into XY problem hell